homeassistant/components/wled/light.py

```diff
@@ -73,6 +74,16 @@ async def async_setup_entry(
         "async_effect",
     )
 
+    platform.async_register_entity_service(
+        SERVICE_PRESET,
```

Comment:
So, this makes the `wled.preset` service, which at this point, recovers/loads a preset. This is nice, however, in the future, we might also want to expose a way to save/create a preset using a service call.

It might be a good idea to take that into account already?

Reply:
I had envisioned having a separate service call for saving a preset (in which case, maybe making this one `wled.load_preset` and adding `wled.save_preset` in the future would make more sense and avoid a breaking change), but I'm also open to doing both under one service call, maybe making the valid values for this service call `entity_id` and then either `load_preset` or `save_preset` as the slot to either load from or save to. I feel like having two separate service calls is a little safer, though, in terms of lessening the chance of accidentally saving over an existing preset.

Reply:
We also have `effect` right now. So I guess this matched that style perfectly.

I guess, we can always add a `save_preset` service or an additional flag to the existing service. Let's stick with this for now 👍